ddcz/urls.py

```diff
@@ -65,7 +66,7 @@
     ### User settings
     path("nastaveni/zmena-skinu/", views.change_skin, name="change-skin"),
     ### Info sites
-    path("aktuality/", views.index, name="news"),
+    path("aktuality/", news.list, name="news"),
```

Comment:
Why are we redirecting this to `/aktuality`? Wouldn't it be better to have only `/` here?

Reply:
I'd argue no since in the future, we may want a different landing page on `/`.